### PR TITLE
[MRG] Add SPIDER_LOADER_WARN_ONLY to toggle between spiderloader failure or warning

### PR DESCRIPTION
**Pull request by redapple**
Related to https://github.com/scrapy/scrapy/pull/2433#issuecomment-281377231

### CODE REVIEW

**Comment by redapple** [comment]
@vshlapakov , would this change work for you?

File: scrapy/settings/default_settings.py [hunk 1]
```diff
@@ -250,6 +250,7 @@
 SCHEDULER_PRIORITY_QUEUE = 'queuelib.PriorityQueue'
 
 SPIDER_LOADER_CLASS = 'scrapy.spiderloader.SpiderLoader'
+SPIDER_LOADER_WARN_ONLY = False
```
Comment (kmike) [inline scrapy/settings/default_settings.py]: Could you please add docs for this option? Or are you waiting for @vshlapakov's feedback?

**Comment by redapple** [comment]
Sorry, I'll add some doc.
But yes, I was also asking if this solution would work (name of option still up of course)

**Comment by vshlapakov** [comment]
@redapple The changes looks good to me, thanks Paul! cc @chekunkov

**Comment by chekunkov** [comment]
LGTM

File: scrapy/settings/default_settings.py (hunk 1 above)
Comment (redapple) [inline scrapy/settings/default_settings.py]: @kmike , docs updated.

**Comment by redapple** [comment]
phew, https://github.com/scrapy/scrapy/pull/2632/commits/9628a739723983f2d3c4079d228dcaa79f558e73 was the whole point of this... ;)

**Comment by kmike** [comment]
Thanks @redapple, @vshlapakov and @chekunkov!

**Pull request merged**